packages/compass-aggregations/src/components/settings/settings.jsx

```diff
@@ -16,7 +16,6 @@ class Settings extends PureComponent {
     isExpanded: PropTypes.bool.isRequired,
     limit: PropTypes.number.isRequired,
     largeLimit: PropTypes.number.isRequired,
-    maxTimeMS: PropTypes.number.isRequired,
```

Comment:
Drive by clean up that cloud also flagged as issue (it's not really causing any issues, just a react warning, we just forgot to clean this up)